contents/data_vis.md

```diff
@@ -523,12 +541,193 @@ Next, we will dive into how to manage and control **layouts**.
+> - `hidexdecorations!(ax; kwargs...)`
+> - `hideydecorations!(ax; kwargs...)`
+> - `hidespines!(ax; kwargs...)`
+
+Remember, we can always ask for help to see what kind of arguments we can use, e.g.,
+
+```jl
+s = """
+    help(hidespines!)
+    """
+sco(s)
+```
+
+Or, for decorations
+
+```jl
+s = """
+    help(hidedecorations!)
+    """
+sco(s)
+```
+
+For elements that **you don't want to hide**, just pass them with `false`, i.e. `hideydecorations!(ax, ticks = false, grid = false)`.
+
+
+Synchronizing your `Axis` is done via
+
+> - `linkaxes! linkyaxes! linkxaxes!`
+>
+> This could be useful when shared axis are desired. Another way of getting shared axis will be by setting `limits!`.
+
+Setting `limits` at once or independenly for each axis is done by calling 
+
+> - `limits!(ax; l,r,b,t)`
+> You can also do `ylims!(low, high)` or `xlims!(low, high)`, and even open ones by doing `ylims!(low=0)` or `xlims!(high=1)`.
+
+Now, the example:
+
+```jl
+@sco JDS.complex_layout_double_axis()
+```
+
+So, now our `Colorbar` needs to be horizontal and the bar ticks be in the lower part. 
+This is done by setting `vertical = false` and `flipaxis = false`. 
+Also, note that we can call many `Axis` into `fig`, or even `Colorbar`'s and `Legend`'s, and then afterwards build the layout. 
+
+Another common layout is a grid of squares for heatmaps. See next example. 
+
+```jl
+@sco JDS.squares_layout()
+```
+
+Where, all labels are in the **protrusions** and each `Axis` has an `AspectData()` ratio. 
+The `Colorbar` is located in the third column and extpands from row 1 up to row 2. 
+
+The next case uses the so called `Mixed()` **alignmode**, which is specially useful when dealing with large empty spaces between `Axis` due to long ticks. 
+
+```jl
+@sco JDS.mixed_mode_layout()
+```
+
+Here, the argument `alignmode =Mixed(bottom=0)` it's shifting the bounding box to the bottom, so that this will align with the panel on the left filling the space. 
+
+Also, see how `colsize!` and `rowsize!` are being used for different columns and rows. 
+You could also put a number instead of `Auto()` but then everything will be fixed. 
+And, additionally, one could also give a `height` or `width` when defining the `Axis`, as in `Axis(fig, heigth = 50)` which will be fixed as well.
+
+#### Nested `Axis` (_subplots_)
+
+It is also possible to define a set of `Axis` (_subplots_) explicitly, and use it to build a main figure with several rows and columns. 
+For instance, the following its a "complicated" arrangment of `Axis`
+
+```jl
+@sc nested_sub_plot!(fig)
+```
+
+which, when used to build a more complex figure by doing several calls, we obatain
+
+```jl
+@sco JDS.main_figure()
+```
+
+Note that, different subplot functions can be call here. Also, each `Axis` here is an independ part of `Figure`. 
+So that, if you need to do some `rowgap!`'s or `colsize!`'s opperations, you will need to do it in each one of them independenly or to all of them together. 
```

Comment:
```suggestion
So that, if you need to do some `rowgap!`'s or `colsize!`'s operations, you will need to do it in each one of them independently or to all of them together. 
```